### PR TITLE
Issue 1527: Updating TANF-app/doc/User-Experience/

### LINKED ISSUE #1527: Update TANF-app/doc/User-Experience/
### Context
After #1515 is merged, `TANF-app/doc/User-Experience/` subdirectory markdowns need to be reviewed and updated. 

### ACs:
- [x] Update existing [README.md](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/README.md) w/ TOC
- [x] This subdir needs a deeper dive audit. There is some outdated information in [here](https://github.com/raft-tech/TANF-app/tree/raft-tdp-main/docs/User-Research)
- [x] Update any broken links within markdowns
- [x] Create subdirectories for research and design, separately

### Tasks:
- [x] Create [hackmd](https://hackmd.io/v1R5-te1Qlq-Of3xpZc6mg) for subdir `README.md` w/ TOC (`sreedevip`)
- [x] Audit the User-Research subdir and determine what is outdated and can be removed (`reitermb`)
 -  **Drop**
    - [x]  `Open research questions.md`
    - [x]  `Overview of Research Activities.md`
    - [x]  `TANF-app/docs/User-Experience/2020, Winter - User Flow Validation/`
    - [x]  `TANF-app/docs/User-Experience/2020, Fall - Concept Prototypes/` 
    - [x]   `TANF-app/docs/User-Experience/Design/` 
- **Move**
  - [x] promote `Design-Principles.md` and `Style-Guide.md` (see below) one level up  into `TANF-app/docs/User-Experience/`
  - [x]  all research syntheses mds  in `TANF-app/docs/User-Experience/` to `TANF-app/docs/User-Experience/Research-Syntheses/` 
  - [x]  contents of `TANF-app/docs/User-Experience/2020, Fall - Concept Prototypes/` to `TANF-app/docs/User-Experience/Research-Syntheses/` and reference them in `2020,Fall Understanding STT roles, source of truth and metadata.md`
- **Change**
  - [x] readme should be converted to table of contents
  - [x] `Design-Guide.md` to `Style-Guide.md` and remove [`Code Docs` section](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Design/Design-Guide.md#code-docs)
  - [x] `TANF-app/docs/User-Experience/Research/` to  `TANF-app/docs/User-Experience/Research-Syntheses/` 
  - [x]  `Stakeholders-and-Personas.md` to `2020, Summer - Understanding Stakeholders and creating personas.md`
  - [x] `2021, Spring - Evil User Journeys & TANF Data Errors (Round 6).md` to `2021, Spring - Evil User Journeys & TANF Data Errors.md`
  - [x] `Winter 2021, Round 4 Synthesis.md` to `2020, Fall - Understanding STT and OFA painpoints.md`
  - [x] include link to [hackmd version of `ST Workshop Facilitation Guide.md`](https://hackmd.io/0zVjk29lSMyngnzdgpZgqg) in `2020, Fall - Understanding STT and OFA painpoints.md`
  - [x]  `STT Attribute Guide` to `2020, Fall - STT Attribute Guide`
 - **Add**
   - [x] `UX-markdown-practices.md` and include [`Code Docs` section of `design-guide.md`](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Design/Design-Guide.md#code-docs)
   - [x]  `readme.md` to `TANF-app/docs/User-Experience/Research-Syntheses/` 
   - [x]   `TANF-app/docs/User-Experience/Dev-Ready-Designs/`

- [x] Open draft PR with subdirectory updates (`adpennington`)
- [x] Review and update any broken links (`valcollignon`, `sreedevip`)

### Notes:
@sreedevip and @reitermb: Please be sure to let @ADPennington know which files should be research and which should be design. Once Alex moves things where they need to go, Miles and Sreedevi can update links in PR.

### PR DESCRIPTION
**Pull request by ADPennington**
## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #1527 

_Acceptance criteria as stated in the issue_
### ACs:
- [x] Update existing [README.md](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/README.md) w/ TOC
- [x] Audit changes implemented
- [x] Update any broken links within markdowns
- [x] Create subdirectories for research and design, separately

### Tasks (audit changes requested):
 -  **Drop**
    - [x]  `Open research questions.md`
    - [x]  `Overview of Research Activities.md`
    - [x]  `TANF-app/docs/User-Experience/2020, Winter - User Flow Validation/`
    - [x]  `TANF-app/docs/User-Experience/2020, Fall - Concept Prototypes/` 
    - [x]  `TANF-app/docs/User-Experience/Design/` 
    
- **Move**
  - [x] `Design-Principles.md` and `Style-Guide.md` promoted  into [`TANF-app/docs/User-Experience/`](https://github.com/raft-tech/TANF-app/tree/4b016c2b4580c048a09e0b450ab4c5ed69d0b46b/docs/User-Experience)
  - [x]  all research syntheses  renamed and moved to [`TANF-app/docs/User-Experience/Research-Syntheses/`](https://github.com/raft-tech/TANF-app/tree/618525aa07a1fbe1ca68bde80392fa050cbe93b0/docs/User-Experience/Research-Syntheses)
  - [x]  contents of `TANF-app/docs/User-Experience/2020, Fall - Concept Prototypes/` now located in [`TANF-app/docs/User-Experience/Research-Syntheses/`](https://github.com/raft-tech/TANF-app/tree/4b016c2b4580c048a09e0b450ab4c5ed69d0b46b/docs/User-Experience/Research-Syntheses) and referenced in [`2020,Fall Understanding STT roles, source of truth and metadata.md`](https://github.com/raft-tech/TANF-app/blob/4b016c2b4580c048a09e0b450ab4c5ed69d0b46b/docs/User-Experience/Research-Syntheses/2020%2C%20Fall%20-%20Understanding%20STT%20Roles%2C%20Source%20of%20Truth%2C%20and%20Metadata.md)
- **Change**
  - [x] [readme](https://github.com/raft-tech/TANF-app/blob/4b016c2b4580c048a09e0b450ab4c5ed69d0b46b/docs/User-Experience/README.md) converted to TOC
  - [x] [`Design-Guide.md` renamed to `Style-Guide.md`](https://github.com/raft-tech/TANF-app/blob/4b016c2b4580c048a09e0b450ab4c5ed69d0b46b/docs/User-Experience/Style-Guide.md) and no longer includes `Code Docs` section
  - [x] [`2020, Fall - Understanding STT and OFA painpoints.md`](https://github.com/raft-tech/TANF-app/blob/4b016c2b4580c048a09e0b450ab4c5ed69d0b46b/docs/User-Experience/Research-Syntheses/2020%2C%20Fall%20-%20Understanding%20STT%20and%20OFA%20painpoints.md) now includes hackmd version of `ST Workshop Facilitation Guide.md`
  
 - **Add**
   - [x] [`UX-markdown-practices.md`](https://github.com/raft-tech/TANF-app/blob/618525aa07a1fbe1ca68bde80392fa050cbe93b0/docs/User-Experience/UX-markdown-practices.md) now includes [`Code Docs` section of `design-guide.md`](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/User-Experience/Design/Design-Guide.md#code-docs)
   - [x]  [`readme.md`](https://github.com/raft-tech/TANF-app/blob/618525aa07a1fbe1ca68bde80392fa050cbe93b0/docs/User-Experience/Research-Syntheses/README.md) added to `TANF-app/docs/User-Experience/Research-Syntheses/` 
   - [x]  [`TANF-app/docs/User-Experience/Dev-Ready-Designs/`](https://github.com/raft-tech/TANF-app/blob/9465af6b5a90927673f24b9cb628640c3d9a835b/docs/User-Experience/Dev-Ready-Designs/README.md)

### Note:

- reference links to files that have moved are broken until PR merged
- UX links in background dir also updated.

### CODE REVIEW

**Review by valcollignon — APPROVED** [review]
LGTM 🚀

**Review by sreedevip — COMMENTED** [review]
Where will the Mural links go?

**Comment by ADPennington** [comment]
> Where will the Mural links go?

@sreedevip these are still there just hidden per yesterday's ux sync: https://github.com/raft-tech/TANF-app/blob/618525aa07a1fbe1ca68bde80392fa050cbe93b0/docs/User-Experience/Research-Syntheses/README.md#research-mural-boards

**Review by sreedevip — APPROVED** [review]

**Comment by abottoms-coder** [comment]
@ADPennington Why does this have both review labels?

**Review by reitermb — APPROVED** [review]
Looks good!

**Comment by lfrohlich** [comment]
Have these been moved or are they hosted elsewhere? 
https://github.com/HHS/TANF-app/tree/main/docs/User-Experience/Design/design-assets/research-artifacts

**Comment by lfrohlich** [comment]
This look good to me, just have two questions ^ before I think it's ready to go.  @sreedevip @reitermb @ADPennington

**Comment by ADPennington** [comment]
> Have these been moved or are they hosted elsewhere? https://github.com/HHS/TANF-app/tree/main/docs/User-Experience/Design/design-assets/research-artifacts

@lfrohlich moved inside markdowns according to [this comment](https://github.com/raft-tech/TANF-app/issues/1527#issuecomment-1010023812)

File: docs/User-Experience/Research-Syntheses/2020, Fall - Understanding STT and OFA painpoints.md [hunk 1]
```diff
@@ -34,6 +34,8 @@
 6. What, if any, loose ends exist for OFA MVP which might be revealed by usability testing and design auditing?
 7. What insight might regional managers and OFA staff provide to engaging STT Users in on-going user research? 
 
+**[Workshop Facilitation Guide for Round 4 Research](https://hackmd.io/0zVjk29lSMyngnzdgpZgqg)**
```
[the remark this answers is not in the document]
Comment (ADPennington) [reply docs/User-Experience/Research-Syntheses/2020, Fall - Understanding STT and OFA painpoints.md:37]: > Is this staying in hack.md?

@lfrohlich yes. @reitermb and @sreedevip recommended not managing it in separate md to keep things tidy.

**Review by lfrohlich — APPROVED** [review]
LGTM

**Pull request merged**